### PR TITLE
Typofix in props.md

### PR DESCRIPTION
**Pull request by volrath**
Fixes `.prop` method signature in "Related Methods"

### CODE REVIEW

File: docs/api/ShallowWrapper/props.md [hunk 1]
```diff
@@ -16,6 +16,6 @@ expect(wrapper.props().foo).to.equal(10);
 
 #### Related Methods
 
-- [`.prop() => Object`](prop.md)
+- [`.prop(key) => Any`](prop.md)
```
Comment (ljharb) [inline docs/api/ShallowWrapper/props.md]: both are actually valid, and the argumentless form is more important anyways. Perhaps we should list both here?

Comment (volrath) [inline docs/api/ShallowWrapper/props.md]: I tried `wrapper.prop()` and it returns `undefined`, and it actually [makes sense](https://github.com/airbnb/enzyme/blob/master/src/ShallowWrapper.js#L484) that it does. It's different from `wrapper.props()` which does return an `Object`. Plus the signature in [`prop.md`](https://github.com/airbnb/enzyme/blob/master/docs/api/ShallowWrapper/prop.md) is `.prop(key) => Any`. I'm not sure if I'm missing something here.

Comment (ljharb) [inline docs/api/ShallowWrapper/props.md]: aha, prop vs props :-) this is a good fix, thanks

Comment (volrath) [inline docs/api/ShallowWrapper/props.md]: yw :)

**Pull request merged**